charts/dbildungscloud-iam-keycloak/templates/deployment.yaml

```diff
@@ -39,6 +42,28 @@ spec:
                   key:  admin-password
             - name: KC_PROXY
               value: "edge"
+            - name: DB_BASE_URL
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.secrets.name }}
+                  key: db.url.jdbc
+            - name: DB_NAME
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.secrets.name }}
+                  key: db.name
+            - name: KC_DB_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.secrets.name }}
+                  key: db.username
+            - name: KC_DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.secrets.name }}
+                  key: db.password
```

Comment:
please use db-password, this secret must also be appended to the 1P Item, let us talk about what do we expect there